File: Code-Java/src/codejava/base/cctype/CtypeEntry.java
package codejava.base.cctype;

public class CtypeEntry {
  private final static char CONST_CHAR = ' ';

  public static void CTypeProgram() {
    boolean b1 = Character.isLetterOrDigit(CONST_CHAR);
    boolean b2 = Character.isLetter(CONST_CHAR);
    boolean b3 = Character.isLowerCase(CONST_CHAR);
    boolean b4 = Character.isUpperCase(CONST_CHAR);
    boolean b5 = Character.isDigit(CONST_CHAR);
    boolean b6 = !(Character.digit(CONST_CHAR, 16) == -1);
    boolean b7 = Character.isISOControl(CONST_CHAR);
    boolean b8 = Character.isWhitespace(CONST_CHAR);
    boolean b9 = Character.isSpaceChar(CONST_CHAR);
    boolean b10 = !Character.isISOControl(CONST_CHAR);
    boolean b11 = !Character.isISOControl(CONST_CHAR) && !Character.isSpaceChar(CONST_CHAR);
    boolean b12 = Character.toString(CONST_CHAR).matches("\\p{Punct}");
    char c1 = Character.toLowerCase(CONST_CHAR);
    char c2 = Character.toUpperCase(CONST_CHAR);
  }
}

File: Code-CSharp/Base/CCType/CTypeEntry.cs
namespace CodeCsharp.Base.CCType; 

public class CTypeEntry {
  private static readonly char ConstChar = ' ';

  public static void CTypeProgram() {
    bool b1 = Char.IsLetterOrDigit(ConstChar);
    bool b2 = Char.IsLetter(ConstChar);
    bool b3 = Char.IsLower(ConstChar);
    bool b4 = Char.IsUpper(ConstChar);
    bool b5 = Char.IsDigit(ConstChar);
    bool b6 = Uri.IsHexDigit(ConstChar);
    bool b7 = Char.IsControl(ConstChar);
    bool b8 = Char.IsWhiteSpace(ConstChar);
    bool b9 = Char.IsSeparator(ConstChar);
    bool b10 = !Char.IsControl(ConstChar);
    bool b11 = !Char.IsControl(ConstChar) && !Char.IsSeparator(ConstChar);
    bool b12 = Char.IsPunctuation(ConstChar);
    char c1 = Char.ToLower(ConstChar);
    char c2 = Char.ToUpper(ConstChar);
  }
}